Automattic/Gravatar-SDK-iOS
# Gravatar OAuth in SwiftUI

### Description
This PR implements the Gravatar OAuth flow.

![CleanShot 2024-08-22 at 14 51 23](https://github.com/user-attachments/assets/cfebb657-3b16-49f3-9ad4-5532e18bd426)


### Testing Steps

- Build and run the SwiftUI demo app
- Go to Profile Editor with oauth demo screen.
- insert an email and open the editor.
  - Check that OAuth flow works as expected.
